Doc/c-api/init_config.rst

```diff
@@ -197,12 +197,12 @@ PyPreConfig
 
    Function to initialize a preconfiguration:
 
-   .. c:function:: void PyPreConfig_InitIsolatedConfig(PyPreConfig *preconfig)
+   .. c:function:: void PyPreConfig_InitPythonConfig(PyPreConfig *preconfig)
 
       Initialize the preconfiguration with :ref:`Python Configuration
       <init-python-config>`.
 
-   .. c:function:: void PyPreConfig_InitPythonConfig(PyPreConfig *preconfig)
+   .. c:function:: void PyPreConfig_InitIsolatedConfig(PyPreConfig *preconfig)
 
       Initialize the preconfiguration with :ref:`Isolated Configuration
       <init-isolated-conf>`.
```

Comment:
nitpick: I would prefer to exchange the description but keep the function in the alphabetical order :-)

Reply:
anti-nitpick: this is consistent with the order for PyConfig functions later in the same file :-)
https://github.com/python/cpython/pull/21198/files#diff-37eacc7b206686c260b26a4fa52ebde4L347

I also like that the more common use case appears first.